app/models/concerns/twitch_user.rb

```diff
@@ -5,9 +5,20 @@ module TwitchUser
   extend ActiveSupport::Concern
 
   included do
-    def follows
-      Rails.cache.fetch([:db, :follows, self], expires_in: 1.hour) do
-        User.where(twitch_id: Twitch::Follows.followed_ids(twitch_id)).joins(:runs).group('users.id')
+    def sync_twitch_follows!
+      ActiveRecord::Base.transaction do
+        current_followed_users = User.where(twitch_id: Twitch::Follows.followed_ids(twitch_id))
+        old_followed_users = twitch_followed_users
+
+        (current_followed_users - old_followed_users).each do |u|
+          TwitchUserFollow.create!(from_user: self, to_user: u)
+        end
```

Comment:
Currently the follow retrieval code in `lib/twitch.rb` only retrieves the first 100 followers since it doesn't iterate down the cursor provided.  Would you like me to add support for this as well when adding the import stuff?

Reply:
Oh damn. Yeah that would be amazing to have if you can add it.